src/platform/plugins/shared/dashboard/server/usage/dashboard_telemetry_collection_task.ts

```diff
@@ -94,23 +95,25 @@ export function dashboardTaskRunner(logger: Logger, core: CoreSetup, embeddable:
     return {
       async run() {
         let dashboardData = getEmptyDashboardData();
-        // const controlsCollector = controlsCollectorFactory(embeddable);
         const processDashboards = (dashboards: DashboardSavedObjectAttributesAndReferences[]) => {
           for (const dashboard of dashboards) {
             // TODO is this injecting references really necessary?
             // const attributes = injectReferences(dashboard, {
             //   embeddablePersistableStateService: embeddable,
             // });
 
-            // dashboardData = controlsCollector(dashboard.attributes, dashboardData);
             dashboardData = collectDashboardSections(dashboard.attributes, dashboardData);
 
             try {
               const panels = JSON.parse(
                 dashboard.attributes.panelsJSON as string
               ) as unknown as SavedDashboardPanel[];
-
               collectPanelsByType(panels, dashboardData, embeddable);
+
+              const controls = JSON.parse(
+                dashboard.attributes.controlGroupInput?.panelsJSON as string
+              ) as unknown as StoredControlGroupInput['panels'];
+              collectStickyControls(controls, dashboardData, embeddable);
```

Comment:
I figured it was fine to throw this `JSON.parse` in the same `try ... catch` block as the panels 🤷 Not sure if we necessarily need a custom error message